src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,67 @@
+package calculator;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Calculator {
+	public String[] splitStr(String calStr){
+		return calStr.split(" ");
+	}
+
+	public boolean isOp(String op){
```

Comment:
```suggestion
	private boolean isOp(String op){
```

내부에서만 사용하는 메서드는 private 으로 해주세요.